.circleci/config.yml

```diff
@@ -0,0 +1,40 @@
+version: 2
+jobs:
+  build_and_test:
+    docker:
+      - image: circleci/node:10-browsers
+    steps:
+      - checkout
+      - run:
+          name: Check for missing change logs
+          command: |
+            git fetch origin master:refs/remotes/origin/master -a
+            node common/scripts/install-run-rush.js change --verify
+      - restore_cache:
+          key: dependency-cache-{{ checksum "common/config/rush/shrinkwrap.yaml" }}
+      - run:
+          name: Install dependencies
+          command: node common/scripts/install-run-rush.js install
+      - save_cache:
+          key: dependency-cache-{{ checksum "common/config/rush/shrinkwrap.yaml" }}
+          paths:
+            - common/temp
+      - run:
+          name: Build source
+          command: node common/scripts/install-run-rush.js rebuild --verbose
+      # - run:
```

Comment:
Is this commented out on purpose?

Reply:
Yes, I'll add a task to enable linting on all repos.